src/runtime/pkg/direct-volume/utils.go

```diff
@@ -0,0 +1,95 @@
+package volume
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"time"
+)
+
+const (
+	mountInfoFileName        = "mountInfo.json"
+	defaultTimeout = 3 * time.Second
+)
+var kataDirectVolumeRootPath = "/run/kata-containers/shared/direct-volumes"
+
+// MountInfo contains the information for the Kata agent to mount a filesystem on the volume inside the guest VM.
+type MountInfo struct {
```

Comment:
Is there a scenario where we'd want to handle different Volume types? For example, block device, network attached, etc. ? May be good to have a VolumeType field?

@bergwolf WDYT -- I recall this feedback from initial PRs.

Reply:
Yes, we should have a volume type there, even if only block types are handled at first, so that we won't break APIs when adding other volume types in the future.

Reply:
added `volumeType` field